components/net/filemanager_thread.rs

```diff
@@ -489,6 +512,103 @@ impl FileManagerStore {
         )
     }
 
+    fn set_headers(&self, response: &mut Response, content_length: u64, mime: Mime,
```

Comment:
No need to make this a method, since `self` is never used. Let's take `&mut HeaderMap` instead of the full response, too.